ca_gazebo/worlds/empty.world

```diff
@@ -1,42 +1,13 @@
-<?xml version="1.0"?>
-<sdf version="1.6">
+<?xml version="1.0" ?>
+<sdf version="1.5">
```

Comment:
Why did you change this line? I don't think it's really necessary?

Reply:
Reverted.